harness/Adapter/index.tsx

```diff
@@ -62,8 +62,8 @@ const styles = (theme) => ({
 
 const inputOptions = [{
   value: 'none',
-  primary: 'No Input'
-}, {
+  primary: 'No Value'
```

Comment:
I prefer 'No Input' here since it provides no Input tool.

Reply:
That makes sense